src/test/util/mining.cpp

```diff
@@ -74,10 +74,12 @@ CTxIn MineBlock(const NodeContext& node, const CScript& coinbase_scriptPubKey)
     return CTxIn{block->vtx[0]->GetHash(), 0};
 }
 
-std::shared_ptr<CBlock> PrepareBlock(const NodeContext& node, const CScript& coinbase_scriptPubKey)
+std::shared_ptr<CBlock> PrepareBlock(const NodeContext& node, const CScript& coinbase_scriptPubKey, bool check)
 {
+    BlockAssembler::Options options;
```

Comment:
not sure if it matters, but previously this was (implicitly) calling `DefaultOptions`, now it calls `Options`.

Reply:
Added an `ApplyArgsManOptions` so we can both manually manipulate the options and get the config from `ArgsManager`.